src/app/screens/LNURLChannel/index.tsx

```diff
@@ -55,7 +58,10 @@ function LNURLChannel() {
         throw new Error(`Failed to call callback: ${callbackResponse.message}`);
       }
 
-      setSuccessMessage(`${t("success")}. ${details.k1} ${nodeId}`);
+      setSuccessMessage(
+        t("success", { name: origin ? origin.name : details.domain })
+      );
+      setResponse(`${details.k1} ${nodeId ? nodeId : `[no nodeId available]`}`);
```

Comment:
Is the nodeId supposed to be null here?

Reply:
when a node id is not available this channel thing does not work. 
so I don't think that this message should ever happen and thus should be needed.

Reply:
In this case it might just be related to testing it with fiatjef?

Reply:
so nodeId comes from line 47. where we do the getInfo call. 
we request the information of the user's connected node and get the node ID from that.

that node id is sent to the callback URL - the service send opens a channel with the user's node (that's why the nodeId is needed).

this whole channel thing works only if there is a node ID available. 
it must never be null. so this `setResponse` does not make sense. 


you could add a check above that if nodeId is falsy then it shows an error and does not even try to do the lnurl-channel call.


I can only imagine that you use an Alby account or some custodial account when you run this. That is not your node so you can not open a a channel to it, or rather you would not. 

the getinfo call likely also does not return the nodeid.